docs/README.rst

```diff
@@ -57,3 +57,12 @@ There is a way to recreate the doc automatically when you make changes, you
 need to install watchdog (``pip install watchdog``) and then use::
 
     make watch
+
+Alternative method using tox
+----------------------------
+
+To compile the documentation to HTML run the following command from this dir::
+
+    tox -e docs
+
+Documentation will be generated (in HTML format) inside the ``../.tox/docs/tmp/html`` dir.
```

Comment:
It's not necessary to be inside the `docs` directory build the documentation with `tox`.

Reply:
Oh yeah, thanks. Fixed now.